stack/packaging-third-party-eups-dependencies.rst

```diff
@@ -2,12 +2,20 @@
 Distributing Third-Party Packages with EUPS
 ###########################################
 
-This page documents how to make a third-party software package install-able using the :command:`eups distrib install` command.
+This page documents how to make a third-party software package installable using the :command:`eups distrib install` command via two methods: the "forked repo" and the "tarball-and-patch" (TaP) package.
 
-Generally, most dependencies :doc:`should be added via conda </stack/conda>` using the ``rubin-env`` metapackage.
-The primary reason to add an eups tarball-and-patch (TaP) package as described here is if we require a fix that has not been released in conda-forge (or if the package has no conda-forge recipe at all, and we do not want to create one).
+Generally, most dependencies :doc:`should be added via conda </stack/conda>` using the ``rubin-env`` metapackage rather than either of these eups methods.
+If the package is already in conda-forge, it can be added to ``rubin-env`` :ref:`via RFC <third-party-approval>`.
+(Changes to ``rubin-env`` are made approximately 2-4 times a year.)
+If the package is not already in conda-forge, `creating and maintaining a conda-forge recipe`_ is about the same level of effort as maintaining a TaP package, and it makes the package available to the conda-forge community at large rather than just LSST Science Pipelines consumers.
```

Comment:
Maybe it's worth mentioning the rationale for choosing a fork instead here, before the whole fork section? Also because forked repos can be standalone/pip/PyPI installable so the "wider distribution" point is not so much in favour of conda-forge.

Reply:
That's what the very next paragraph was supposed to do.

Reply:
I will invert the phrasing in the topic sentence to make it clear.